### DISCUSSION TITLE
Single image/photo square interface

### DISCUSSION THREAD

**Discussion by IlyaPetrovM** [directus/directus#9145] [Archive]
Hi!
How to make square-shaped interface for single photo like in `directus_users` collection?

**Comment by joselcvarela** [comment]
Hello @IlyaPetrovM 
Do you want to replace the round-shaped interface in user detail page, by a square-shaped?
I don't think this is possible as of right now.

![image](https://user-images.githubusercontent.com/14039341/138819505-af7f6e03-cc41-4964-87a6-f84b46e5d469.png)

**Comment by IlyaPetrovM** [comment]
Well, solution found. I changed `.image-interface` class in custom styles
